sr_blockly_blocks/blocks/initialise.js

```diff
@@ -2,7 +2,8 @@ Blockly.Blocks['initialise'] = {
   init: function() {
     this.appendDummyInput()
         .appendField("Load Commander")
-        .appendField(new Blockly.FieldDropdown([["Arm Commander", "arm_commander"], ["Hand Commander", "hand_commander"], ["Hand and Arm", "hand_and_arm"]]), "commander")
+        .appendField(new Blockly.FieldDropdown([["Arm Commander", "arm_commander"], ["Hand Commander", "hand_commander"],
+          ["Hand and Arm", "hand_and_arm"], ["Robot Commander", "robot_commander"]]), "commander")
```

Comment:
I know this wasn't your error but could you change "Hand and Arm" to "Hand and Arm Commander"?